recipes/mizuroute/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "1.2" %}
+
+package:
+  name: mizuroute
+  version: {{ version }}
+
+source:
+  url: https://github.com/ncar/mizuroute/archive/v{{ version }}.tar.gz
+  sha256: 2238525e2f43e2d4fd27021a222477c9841c410d4d3a7330a02e8bc101b64f57
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('fortran') }}
+  host:
+    - netcdf-fortran
+  run:
+    - netcdf-fortran
+
+test:
+  commands:
+    - which route_runoff.exe
```

Comment:
Can you try running a help or version command?

Reply:
mizuRoute currently doesn't have a help or version command - this might be added in a future release though.

Reply:
Can you use existence tests here instead of which please? https://conda-forge.org/docs/maintainer/adding_pkgs.html#simple-existence-tests

Reply:
Done